pkg/scraper/bruneida_scraper.go

```diff
@@ -66,8 +76,17 @@ func ScrapeBruneida() []model.Job {
 		linkCollector.Visit(url)
 	}
 
-	jobCollector.Visit("https://www.bruneida.com/FEMALE-SALES-ASSISTANT-106578")
+	// Adding links to the existing jobs slice
+	for i := range jobs {
+		jobs[i].Link = links[i]
+		jobs[i].JobId = jobIds[i]
+	}
```

Comment:
I think this is not the best way to be doing this due to the point I mentioned in the other scraper.

What you can do instead is create a job map with the job id as the key, then access that map to modify the job when needed.

E.g. very rough code
```go
jobMap := map[string]model.Job{}

.....

linkCollector.OnHTML(func.....
    getJobId
    job := .....
    jobMap[jobid] = job
    Visit(url)
)


linkCollector.OnHTML(func.....
    getJobId // probably a way to get current page url with colly?
    job, ok :=  jobMap[jobid] // probably want to check if it's ok, and what to do if it's not ok (e.g. fill it the other values, or just ignore)
    
    jobMap[jobid] = job
)
```

Reply:
```go
linkCollector.OnHTML(func.....
    getJobId // probably a way to get current page url with colly?
    job, ok :=  jobMap[jobid] // probably want to check if it's ok, and what to do if it's not ok (e.g. fill it the other values, or just ignore)
    
    jobMap[jobid] = job
)
```

The url can only be obtained in onRequest method AFAIK